README.md

```diff
@@ -172,6 +173,8 @@ const x = codegen.require('./es6-identity', 3)
 const x = 3
 ```
 
+> You can also use `codegen.import` as an alias for `codegen.require` 🔥
```

Comment:
I would actually prefer to NOT have this alias. We should chose one or the other to reduce the surface area of the API and confusion of the library. I prefer `codegen.require`.